File: Tema8_2_Java/src/exemples/Prova1.java
package exemples;
import org.bson.Document;

import com.mongodb.MongoClient;
import com.mongodb.client.MongoDatabase;

public class Prova1 {

    public static void main(String[] args) {
        MongoClient con = new MongoClient("localhost" , 27017);
        MongoDatabase bd = con.getDatabase("test");
        
        Document doc = new Document();
        doc.put("msg4", "Missatge inserit des de Java");
        bd.getCollection("exemple").insertOne(doc);
        
        con.close();
    }
}

File: Tema8_2/src/exemples/Prova1.kt
package exemples


import org.bson.Document
import com.mongodb.MongoException
import com.mongodb.MongoClient

//import com.mongodb.MongoClientURI


fun main(args: Array<String>) {
        val con = MongoClient("localhost" , 27017);
        val bd = con.getDatabase("test");
        
        val doc = Document();
        doc.put("msg4", "Missatge inserit des de Kotlin");
        bd.getCollection("exemple").insertOne(doc);
        
        con.close();

}


